acl_setup.php

```diff
@@ -236,7 +258,7 @@
  $gacl->add_acl(
   array(
    'acct'=>array('bill', 'disc', 'eob', 'rep', 'rep_a'),
-   'admin'=>array('calendar', 'database', 'forms', 'practice', 'superbill', 'users', 'batchcom', 'language', 'super', 'drugs', 'acl'),
+   'admin'=>array('calendar', 'database', 'forms', 'practice', 'superbill', 'users', 'batchcom', 'language', 'super', 'drugs', 'acl','multipledb','gadd','gcalendar','glog'),
```

Comment:
Note gadd, gcalendatr, and glog are not in the admin section (note they are in the groups section).
Plus you did not add these items to the admin acl in the upgrade script.